right/src/config_parser/basic_types.c

```diff
@@ -26,6 +27,16 @@ uint32_t ReadUInt32(config_buffer_t *buffer)
     return uInt32;
```

Comment:
Is the above correct?

My brain says that 0xDDCCBBAA in little endian gets parsed as 0x0000DDAA.

Reply:
It seems like ReadUInt32 is used nowhere in the project, which explains why this was not noticed earlier...

Fixing it to expect lower endian...